module/items/spell.ts

```diff
@@ -84,10 +84,6 @@ export class Spell extends BWItem<SpellData> {
         user: User
     ): void {
         super._preCreate(data, options, user);
-        this.updateSource({
-            'system.maxRange': game.i18n.localize('BW.spell.tenPaces'),
-            'system.description': game.i18n.localize('BW.spell.newSpell'),
-        });
```

Comment:
Good catch on this bug (any spells being added to a character will be re-created, and therefore have their starter props set to these localized ones).

I think it would be good to preserve the localization behaviour though for default spells.

Can you expand on this a bit? basically
```
if (this.system.maxRange === defaultSpellRange) { this.updateSource( localize max range ); }
if (this.system.description === defaultSpellDescription) { this.updateSource(localize description); }
```
That way we keep the localization on new spells, but don't wipe out existing compendium data.